directord/server.py

```diff
@@ -598,92 +598,23 @@ def run_interactions(self, sentinel=False):
-                        execution_time=data_item.get("execution_time", 0),
-                        return_timestamp=data_item.get("return_timestamp", 0),
-                        component_exec_timestamp=data_item.get(
-                            "component_exec_timestamp", 0
-                        ),
-                        recv_time=time.time(),
-                    )
-
-                    for new_task in data_item.get("new_tasks", list()):
-                        self.log.debug("New task found: %s", new_task)
-                        if "targets" in new_task and new_task["targets"]:
-                            targets = [i for i in new_task["targets"]]
-                            self.log.debug(
-                                "Using existing targets from old job"
-                                " specification %s",
-                                targets,
-                            )
-                        else:
-                            targets = self.workers.keys()
-                            self.log.debug(
-                                "Targets undefined in old job specification"
-                                " running everwhere"
-                            )
-
-                        # NOTE(cloudnull): If the new task identity is set but
-                        #                  with a null value, reset the value
-                        #                  to that of the known workers.
-                        if "identity" in new_task and not new_task["identity"]:
-                            self.log.debug("identities reset to all workers")
-                            new_task["identity"] = [
-                                i for i in self.workers.keys()
-                            ]
-
-                        if "job_id" not in new_task:
-                            new_task["job_id"] = utils.get_uuid()
-
-                        self.create_return_jobs(
-                            task=new_task["job_id"],
-                            job_item=new_task,
-                            targets=targets,
-                        )
-
-                        for target in targets:
-                            self.log.debug(
-                                "Queuing callback job [ %s ] for identity"
-                                " [ %s ]",
-                                new_task["job_id"],
-                                target,
-                            )
-                            self.send_queue.put(
-                                dict(
-                                    identity=target,
-                                    command=new_task["verb"],
-                                    data=new_task,
-                                )
-                            )
 
             if time.time() > prune_time:
                 self.workers.prune()
                 prune_time = time.time() + 762
 
             if sentinel:
+                self.driver.job_stop()
```

Comment:
the driver spec has job_close, unless we expect messaging to have a specific close method, this should probably just use that. 

https://github.com/cloudnull/directord/blob/main/directord/drivers/__init__.py#L268

Reply:
Done.